mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -86,8 +88,11 @@ CLASS net/minecraft/class_31 net/minecraft/item/ItemStack
 	METHOD method_717 isDamageable ()Z
 	METHOD method_718 equals2 (Lnet/minecraft/class_31;)Z
```

Comment:
Adding a "2" at the end does not say anything about how this method is different from the first, it should have a more descriptive name.

Reply:
i dont even see equals in the class, but if i try to name it equals it says the name is not unique so equals might be reserved by the java Object object

Reply:
Not a part of this PR, can be ignored.